modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionTypeGlobalJS/build.gradle

```diff
@@ -1,15 +1,26 @@
 task runGradleTest
 
 runGradleTest {
-	dependsOn ":global-js-client-extension:createClientExtensionConfig"
+	dependsOn ":global-js-client-extension:createClientExtensionConfig", ":instance-scoped-global-js-client-extension:createClientExtensionConfig"
 
 	doLast {
-		Project clientExtensionProject = project(":global-js-client-extension")
+		File clientExtensionConfigJSONFile = _getClientExtensionConfigJSONFile("global-js-client-extension")
 
-		File clientExtensionConfigJSONFile = clientExtensionProject.file("${clientExtensionProject.buildDir}/liferay-client-extension-build/global-js-client-extension.client-extension-config.json")
+		assert clientExtensionConfigJSONFile.text.contains('scriptElementAttributesJSON={\\"async\\":false,\\"data-bool-false\\":false,\\"data-bool-true\\":true,\\"data-empty\\":\\"\\",\\"data-str-false\\":\\"false\\",\\"data-str-true\\":\\"true\\",\\"defer\\":true,\\"nomodule\\":false}')
 
-		assert clientExtensionConfigJSONFile.exists()
+		clientExtensionConfigJSONFile = _getClientExtensionConfigJSONFile("instance-scoped-global-js-client-extension")
 
-		assert clientExtensionConfigJSONFile.text.contains('scriptElementAttributesJSON={\\"async\\":false,\\"data-bool-false\\":false,\\"data-bool-true\\":true,\\"data-empty\\":\\"\\",\\"data-str-false\\":\\"false\\",\\"data-str-true\\":\\"true\\",\\"defer\\":true,\\"nomodule\\":false}')
+		assert clientExtensionConfigJSONFile.text.contains('restrictedToAdminPages=true')
+		assert clientExtensionConfigJSONFile.text.contains('scope=instance')
+		assert clientExtensionConfigJSONFile.text.contains('scriptLocation=head')
 	}
+}
+private File _getClientExtensionConfigJSONFile(String projectName) {
+	Project clientExtensionProject = project(":${projectName}")
+
+	File clientExtensionConfigJSONFile = clientExtensionProject.file("${clientExtensionProject.getProjectDir()}/build/liferay-client-extension-build/${projectName}.client-extension-config.json")
+
+	assert clientExtensionConfigJSONFile.exists()
+
+	return clientExtensionConfigJSONFile
```

Comment:
Could this one be split up into a couple of cases that utilize `testClientExtensionBuildFileContentsTestCase.gradle` such as [testConfigJson.gradle](https://github.com/liferay/liferay-portal/blob/master/modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionWithVirtualInstanceId/testConfigJson.gradle?plain=1#L11)